pkgs/development/libraries/libraw1394/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation rec {
   meta = { 
     description = "Library providing direct access to the IEEE 1394 bus through the Linux 1394 subsystem's raw1394 user space interface";
     homepage = "https://ieee1394.wiki.kernel.org/index.php/Libraries#libraw1394";
-    license = ["GPL" "LGPL"];
+    license = stdenv.lib.licenses.lgpl2;
```

Comment:
Personally, I don't think we need to specify licensing too precisely, like Debian does. Nixpkgs users mostly just use the binaries built from it, so their redistributability and unrestricted usability seems the main point, which are the vague free/unfree/unfree-redistributable options. (Stuff like patent risks might come as the next point.)